src/app/register/register.component.scss

```diff
@@ -0,0 +1,17 @@
+@import “../variables”;
+
+.registration_box{
+  text-align: center;
+}
+
+.header-1 {
+  color : #04529D;
```

Comment:
`$header-left-color`?

how is this different form the `header-1` in `login.component.scss`?